.github/workflows/run-workflow-production.yml

```diff
@@ -0,0 +1,159 @@
+# This action relies on authenticating the Dataform Command Line Interface (CLI) within Github Actions to use a service account for the relevant GCP project 
+
+# to run queries in BigQuery. It does this via Workload Identity Federation and GCP Application Default Credentials. This means that this action requests a 
+
+# JSON key from the Github identity provider to prove that it really is this Github repository. This key lasts only while this action is running. 
+
+# Instructions to set this up are below. 
+
+# 1. Create a GCP service account called something like your-repo-name-github-actions@rugged-abacus-218110.iam.gserviceaccount.com 
+
+# 2. Give the service account the following GCP IAM roles: BigQuery Basic Custom, Data Editor, Data Viewer, Job User, Resource Viewer, Workload Identity User 
+
+# 3. From the Workload Identity Federation screen in GCP click 'Create Pool' to create a Workflow Identity Provider and Pool. 
+
+#    Give the pool the ID 'github-actions-identity-pool', the provider the ID 'bat-dataform-github-actions' (or similar), 
+
+#    do not upload a JWK file, set audience to Default, ensure Enabled provider is switched on, set the Issuer URL to 
+
+#    https://token.actions.githubusercontent.com and map the following attributes between Google and 
+
+#    the OIDC provider: attribute.repository_owner to assertion.repository_owner, attribute.aud to assertion.aud, attribute.actor to assertion.actor, 
+
+#    google.subject to assertion.sub and attribute.repository to assertion.respository. 
+
+# 4. Click 'Grant access' and give access to the service account you created in (1) when the 'repository' attribute is equal to something 
+
+#    like 'DfE-Digital/bat-dataform' (replace this with the organisation and repo name of your Github repo). 
+
+# 5. Create a file in this Github repo at the top level called .df-credentials.json. Put a JSON like 
+
+#    {"projectId": "rugged-abacus-218110", "location": "europe-west2"} inside it, replacing this projectId with your GCP project name. 
+
+# 6. Edit the 'Authenticate to Google Cloud' step below such that service_account is the one you created in (1), and to change parts of 
+
+#     workload_identity_provider: replace 712009772377 with your GCP project ID (an integer), github-actions-identity-pool with the ID of 
+
+#     your identity pool (if it's different) and bat-dataform-github-actions with the ID of your identity provider. 
+
+# 7. Commit and merge to your main/master branch. 
+
+  
+
+name: CI 
```

Comment:
It looks like this workflow has the same name as the other, is it worth giving them separate names? It sounds like this one is related to continuous deployment rather than continuous integration.